ClassicLootManager/Modules/Auctioning/Common/AuctionInfo.lua

```diff
@@ -47,7 +47,7 @@ end
 
 local function Clear(self)
     for _, item in pairs(self.items) do
-        item:Clear()
+        --item:Clear()
```

Comment:
Why do you remove this one?

Reply:
Upon starting bidding, all items get Cleared, which means that the `total` also gets reset. I'll deal with this in another way.